### PR TITLE
add descriptions about binding source

### PR DESCRIPTION
**Pull request by lawyerphx**
### What is changed, added or deleted? (Required)

- add descriptions about the parameter `source` introduced by https://github.com/pingcap/tidb/pull/17254
- fix https://github.com/pingcap/tidb/issues/16748

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: sql-statements/sql-statement-show-bindings.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ SHOW [GLOBAL | SESSION] BINDINGS [ShowLikeOrWhereOpt];
 | update_time | 更新时间 |
 | charset | 字符集 |
 | collation | 排序规则 |
+| source | 创建方式 |
```
Comment (zz-jason) [inline sql-statements/sql-statement-show-bindings.md:49]: 举个例子吧

Comment (lawyerphx) [reply sql-statements/sql-statement-show-bindings.md]: comment addressed @zz-jason

**Comment by zz-jason** [comment]
@lawyerphx 这个功能应该还没有带到 4.0 中，需要 cherry-pick 下代码

**Comment by zz-jason** [comment]
@lawyerphx 这个功能应该还没有带到 4.0 中，需要 cherry-pick 下代码

**Pull request merged**